docker-entrypoint.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$DATABASE_HOST" ]; then
+  until nc -z -v -w30 "$DATABASE_HOST" 5432
+  do
+    echo "Waiting for postgres database connection..."
+    sleep 1
+  done
+  echo "Database is up!"
+fi
+
+# Apply database migrations
+if [[ "$APPLY_MIGRATIONS" = "1" ]]; then
+    echo "Applying database migrations..."
+    ./manage.py migrate --noinput
+fi
+
+if [[ "$SETUP_DEV_OIDC" = "1" ]]; then
```

Comment:
With this PR, the code blocks:
- SETUP_DEV_OIDC
- GENERATE_OPENID_KEY
- CREATE_SUPERUSER
.. are not executed. Should they be executed?

Reply:
I'm unsure about this but it is something that can be added later if needed.